src/repositories/donationRepository.js

```diff
@@ -4,17 +4,30 @@ const { DonationBridgeStatus, DonationStatus } = require('../models/donations.mo
 const getTotalUsdValueDonatedToCampaign = async (app, { campaignId }) => {
   const donationModel = app.service('donations').Model;
   const result = await donationModel.aggregate([
-    {
-      $match: {
-        campaignId,
+    [
+      {
+        $match: {
+          $or: [
```

Comment:
@aminlatifi  I just chaged this query, it will not have any side effects, so I take responsibility and merge it